pkg/logger/logger.go

```diff
@@ -18,65 +18,67 @@ func InitLogger() {
 	}
 }
 
-type Msg struct {
-	Message string
-	Params  map[string]interface{}
+type message struct {
```

Comment:
Why is this structure private?

Reply:
because we don't need to access this variable from external packages